Library/Homebrew/brew.sh

```diff
@@ -138,6 +143,11 @@ else
   HOMEBREW_GIT="git"
 fi
 
+if [[ -n "$HOMEBREW_DEVELOPER" && -x "$HOMEBREW_GIT_PATH" ]]
```

Comment:
What about putting this as an `elif` above; it feels like if `HOMEBREW_FORCE_BREWED_GIT ` is set this shouldn't override it?